Assets/Scripts/Game/UserInterfaceWindows/DaggerfallTravelPopUp.cs

```diff
@@ -213,25 +193,51 @@ public override void Update()
         //Update when player pushes buttons etc.
         void Refresh()
         {
-            UpdateTogglePanels();
+            UpdateToggles();
             UpdateLabels();
         }
 
-        //Updates the positions for the panels to indicate which button is selected
-        void UpdateTogglePanels()
+        void UpdateToggles()
         {
-            if (speedCautious)
-                speedToggleColorPanel.Position = cautiousPanelPos;
-            else
-                speedToggleColorPanel.Position = recklessPanelPos;
-            if (sleepModeInn)
-                sleepToggleColorPanel.Position = innPanelPos;
-            else
-                sleepToggleColorPanel.Position = campoutPos;
-            if (travelShip)
-                transportToggleColorPanel.Position = shipPos;
-            else
-                transportToggleColorPanel.Position = footPos;
+            if (speedCautious){
+                UpdateToggle(3,67,4,4,toggleColor);            //cautious
+                UpdateToggle(3,57, 4,4,notToggledColor);        //reckless
+            }
+            else{
+                UpdateToggle(3,67,4,4,notToggledColor);
+                UpdateToggle(3,57, 4,4,toggleColor);
+            }
+            if (sleepModeInn){
+                UpdateToggle(3,35,4,4,toggleColor);            //inn
+                UpdateToggle(116, 35, 4,4, notToggledColor);    //camp
+            }
+            else{
+                UpdateToggle(3,35,4,4,notToggledColor);
+                UpdateToggle(116, 35, 4,4, toggleColor);
+            }
+            if (travelShip){
+                UpdateToggle(116, 57, 4,4, toggleColor);       //ship
+                UpdateToggle(116, 67,4,4,notToggledColor);      //foot/horse
+            }
+            else{
+                UpdateToggle(116, 57, 4,4, notToggledColor);
+                UpdateToggle(116, 67,4,4,toggleColor);
+            }
+        }
+
+        //updates texture based on travel options selected
+        void UpdateToggle(int startX, int startY, int width, int height, Color32 color)
```

Comment:
Unfortunately this breaks texture replacement. Imported textures can have higher resolution than original image so we can't change pixels colors in a reliable way. The workaround is to scale images individually and draw them in the correct order, which is what was already being done here.

Reply:
I want to add that you can also draw all checkboxes on a single overlay texture with transparent background, what's important is that native texture is not edited because it can have any resolution and format if imported from mods. See #1180 and #1181.

Reply:
We definitely want to keep the base image modable, thank you for picking this @TheLacus. 👍